application/config.json.template

```diff
@@ -24,7 +24,6 @@
    },
    "wolframAlphaAppId": "79J52T-6239TVXHR7",
    "helpSystem": {
-       "stagingChannelPattern": "ask_here",
```

Comment:
I would like to keep this setting in the config, even if its maybe not used. Just because its part of the help system.

Reply:
(by the way, when you remove something from the config, you would also need to update the actual code `Config.java`)

Reply:
> (by the way, when you remove something from the config, you would also need to update the actual code `Config.java`)

it was in `HelpSystemHelper` no? i did removed it from there

Reply:
Right. overlooked that. anyways, thanks for restoring it.